setup.cfg

```diff
@@ -1,26 +1,56 @@
 [metadata]
 name = grizli
-description = Grism analysis pipeline
+version = 1.6.0
 author = G. Brammer
 author_email = gbrammer@gmail.com
+description = Grism redshift and line analysis software
+long_description = file: README.rst
+url = https://github.com/gbrammer/grizli
 license = MIT
-url = http://github.com/gbrammer/grizli
-edit_on_github = False
-github_project = gbrammer/grizli
+license_files = LICENSE.txt
+project_urls = 
+    Documentation = https://grizli.readthedocs.io/
+    Source = https://github.com/astropy/astropy
```

Comment:
Oh, and this should point to grizli on github, not astropy.  👍

Reply:
Ah, my bad. I was referring from the astropy `setup.cfg` hence the error.

Reply:
I think so.